### DISCUSSION TITLE
Serenity / .net 5 / blazor

### DISCUSSION THREAD

**Discussion by kilroyFR** [serenity-is/Serenity#5306] [Serenity Ideas]
hey there Volkan & Serenity team

with the new .net5 Framework, could you tell which direction you are going to follow (especially interested to know if blazor is gonna be part of the goals in a future updated roadmap ?).
Thanks for let us know please

PS : i have not studied yet blazor/webassembly and am already aware of MS issues with some technologies (experienced this badly with silverlight in the past), but probably waiting to see how it evolves and if community will follow blazor or not

**Comment by VictorTomaili** [comment]
Currently we don't plan any feature with blazor. Also we work on .net5 upgrade, you can see net5 branch for this. 

We enabled discussion for this repository. So you can open questions as discussion. Please use issues for real issues.

**Reply by kilroyFR** [reply]
thanks, yes sure, this discussions tab is new, not having the reflex yet.
